internal/host/hostcommands/install_cmd.go

```diff
@@ -73,7 +73,17 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 		return nil, err
 	}
 
-	fullCmd, err := i.getFullInstallerCommand(cluster, host, bootdevice)
+	// TODO(flaper87): Check if this fails to load
```

Comment:
What's the downside to checking for this error?

I think it's very unlikely that this fails.

Reply:
I'm actually changing this since we are not going to backport this all the way back to 2.3. 

The latest findings show that this may be broken only for master and 2.4. With that, I feel more confident with making a bigger change. 

That said, to answer your question, it does make our tests fail, which gives me 0 confidence on how "unlikely" this scenario could be. :sweat_smile: